### ISSUE TITLE
[examples/cube] Does not support spacenavd running over Unix

### ISSUE THREAD

**Issue by Arxcis** [FreeSpacenav/libspnav#7]
**My zsh output**
```
➜  cube git:(master) ✗ head  /var/log/spnavd.log 
Spacenav daemon v0.7.1-5-g11a43b8
reading config file: /etc/spnavrc
failed to find any supported devices
failed to find any supported devices
failed to find any supported devices
adding device.
device name: 3Dconnexion SpaceNavigator
using device: 3Dconnexion SpaceNavigator (/dev/input/event20)

➜  cube git:(master) make
gcc -pedantic -Wall -g -I../.. -I/usr/local/include   -c -o cube.o cube.c
gcc -pedantic -Wall -g -I../.. -I/usr/local/include   -c -o vmath.o vmath.c
gcc -o cube cube.o vmath.o -L../.. -L/usr/local/lib -lX11 -lGL -lGLU -lm -lspnav

➜  cube git:(master) ✗ ./cube 
failed to connect to the space navigator daemon
```

I have a fix for this issue in my fork. I am preparing a PR..

**Comment by Arxcis** [comment]
Fixed by #8

**Comment by jtsiomb** [comment]
There's no need for the cube example to support both methods. It exists to show how to implement 6dof transformations in a graphics program, based on the events provided by libspnav.
